subsys/bluetooth/controller/Kconfig

```diff
@@ -733,7 +733,6 @@ comment "BLE Controller debug configuration"
 
 config BT_CTLR_ASSERT_HANDLER
 	bool "Bluetooth Controller Assertion Handler"
```

Comment:
Rename to "Application Defined Assertion Handler"

Reply:
Done.